ui/src/app/edge/history/history.module.ts

```diff
@@ -143,6 +146,9 @@ import { TimeOfUseTariffDischargeWidgetComponent } from './timeofusetariffdischa
     SymmetricPeakshavingChartComponent,
     SymmetricPeakshavingChartOverviewComponent,
     SymmetricPeakshavingWidgetComponent,
+    ThermometerChartComponent,
+    ThermometerChartOverviewComponent,
+    ThermometerWidgetComponent,

```

Comment:
wrapping these components into modules is recommended. Improves readability and compactness